tasks/section1.yml

```diff
@@ -362,7 +362,7 @@
       - rule_1.1.4
 
 - name: "SCORED | 1.1.5 | AUDIT | Ensure noexec option set on /tmp partition"
-  command: /bin/true
+  shell: mount | grep /tmp | grep noexec
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*